src/shared/pricing.ts

```diff
@@ -42,20 +41,3 @@ export function calculateLabelValue(label: string): number {
   if (label.toLowerCase().includes("month")) return 5 + (number - 1) * 8;
   return 0;
 }
-
-export function calculateDurations(labels: Label[]): number[] {
```

Comment:
why was this removed?

Reply:
This was to satisfy Knip because this export wasn't used anywhere

Reply:
hmm that's weird

Reply:
Added it back as a comment. If it is supposed to be used somewhere let me know I can have a look.